src/IO/Radio.h

```diff
@@ -47,7 +47,7 @@ class Radio : public IO
 
 	std::shared_ptr<std::ofstream> fileStream = nullptr;
 
-	int fd;
+	int fd{};
```

Comment:
I don't understand this change

Reply:
Ok, it seems it's because we didn't initialize explicitly the variable. Maybe we should change this to something else, like `int fd = 0`?